pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1091,15 +1088,7 @@ public static void updateKit(long dsmKitId, String userId, DDPInstanceDto ddpIns
 
             KitRequestShipping kitRequestShipping = new KitRequestShipping(null, dsmKitId, null, null, null, null);
             kitRequestShipping.setLabelDate(labelDate);
-
-            try {
-                UpsertPainlessFacade.of(DBConstants.DDP_KIT_REQUEST_ALIAS, kitRequestShipping, ddpInstanceDto, ESObjectConstants.DSM_KIT_ID,
-                        ESObjectConstants.DSM_KIT_ID, dsmKitId, new PutToNestedScriptBuilder()).export();
-            } catch (Exception e) {
-                logger.error(String.format("Error updating label date of kit request shipping with dsm kit id: %s in ElasticSearch",
-                        dsmKitId));
-                e.printStackTrace();
-            }
+            upsertUpdatedKitInfoIntoES(ddpInstanceDto, kitRequestShipping, dsmKitId);
```

Comment:
Are we calling upsertUpdatedKitInfoIntoES for Juniper kits? If so, why?

Reply:
I added the check in the method, so that method has a check if there is an index in ES